pkg/operation/botanist/component/kubeapiserverexposure/kube_apiserver_sni_test.go

```diff
@@ -18,6 +18,7 @@ import (
 	"context"
 	"time"
 
+	"github.com/gardener/gardener/pkg/apis/core/v1beta1"
```

Comment:
We typically use an alias here:

```suggestion
	gardencorev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
```